On the HiFive1 the ramdisk is occasionally loaded to the wrong address
range, causing boot to hang during ramdisk loading. This patch fixes
this occasionally occurring issue.

See https://lists.denx.de/pipermail/u-boot/2021-October/463798.html for
details. As per https://lists.denx.de/pipermail/u-boot/2021-October/463844.html
this patch should no longer be needed as soon as v2022.01 is released.

diff -upr u-boot-2021.10.orig/include/configs/sifive-unmatched.h u-boot-2021.10/include/configs/sifive-unmatched.h
--- u-boot-2021.10.orig/include/configs/sifive-unmatched.h	2021-10-15 19:07:43.123472870 +0200
+++ u-boot-2021.10/include/configs/sifive-unmatched.h	2021-10-15 19:08:27.357420781 +0200
@@ -25,6 +25,7 @@
 
 #endif
 
+#define CONFIG_SYS_BOOTMAPSZ		(256 << 20)
 #define CONFIG_SYS_SDRAM_BASE		0x80000000
 #define CONFIG_SYS_INIT_SP_ADDR		(CONFIG_SYS_SDRAM_BASE + SZ_2M)
 
